examples/common/builders.py

```diff
@@ -63,6 +63,12 @@ def build_optimizer(cfg, model):
                               betas=cfg.betas,
                               eps=cfg.eps,
                               weight_decay=cfg.weight_decay)
+    elif cfg.name == 'decoupled_lionw':
+        return DecoupledLionW(model.parameters(),
+                    lr=cfg.get('lr', 1e-4),
+                    betas=cfg.get('betas', (0.9, 0.99)),
+                    weight_decay=cfg.get('weight_decay', 0.0)
```

Comment:
I would prefer not to allow any defaults for optimizer hparams... I want our YAMLs to be forced to specify each hparam so it's easily readable without having to dig into the github commit for a particular run.

```suggestion
                    lr=cfg.lr,
                    betas=cfg.betas,
                    weight_decay=cfg.weight_decay
```